doc/releases/changelog-dev.md

```diff
@@ -76,6 +76,13 @@
 * When given a callable, `qml.ctrl` now does its custom pre-processing on all queued operators from the callable.
   [(#4370)](https://github.com/PennyLaneAI/pennylane/pull/4370)
 
+* `qml.interfaces.set_shots` now has null behaviour for the experimental Device and accepts
```

Comment:
What does it mean by "null behaviour"?

Reply:
Basically, it will not error out but it also won't actually do anything. I could probably come up with some better phrasing.